context/app/api/client.py

```diff
@@ -268,7 +272,7 @@ def get_descendant_to_lift(self, uuid):
                     "must": [
                         {
                             "term": {
-                                "vitessce-hints": "is_support"
+                                "vitessce-hints": "is_image"
```

Comment:
Sorry I know this is still a draft. This will fix the broken dataset visualizations, but will break the publication vignettes. I imagine you'll add the hint as a param or similar?

Reply:
Yes, good callout - as discussed on Slack, this was the approach I took. A new boolean arg has been added and is now used to distinguish calls for publications and image pyramids.